### PR TITLE
feat: check property function for dynamic linked callee contract

### LINKED ISSUE #300: Update "check_wasm" to check callee WASM's having "_get_callable_points_properties"
In static check, WASMs which can be called as a dynamic link callee should check it has `_get_callable_points_properties`.

So we should add this logic to `check_wasm`.

### PR DESCRIPTION
**Pull request by dudong2**
# Description

Closes #300 

## Types of changes

- [ ] Bug fix (changes which fixes an issue)
- [x] New feature (changes which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] ETC (build, ci, docs, perf, refactor, style, test)

## Checklist

- [x] I followed the [contributing guidelines](https://github.com/Finschia/cosmwasm/blob/main/CONTRIBUTING.md).
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] The PR title and commits are followed [conventional commit form](https://www.conventionalcommits.org/en/v1.0.0).

### CODE REVIEW

**Review by zemyblue — COMMENTED** [review]
Please add changelog.

**Comment by loloicci** [comment]
In this repository, the changelog is generated automatically when each new tag is released. So each PR does not need to contain its information in the changelog.

**Review by da1suk8 — APPROVED** [review]

**Comment by loloicci** [comment]
add a test that checks contracts without callable points and `_get_callable_points_properties` passes `check_wasm_exports` to `check_wasm_exports_works`

File: packages/vm/src/compatibility.rs [hunk 1]
```diff
@@ -542,6 +542,21 @@ mod tests {
         let module = deserialize_wasm(&wasm).unwrap();
         check_wasm_exports(&module).unwrap();
 
+        // valid
+        let wasm = wat::parse_str(
+            r#"(module
+                (type (func))
+                (func (type 0) nop)
+                (export "_get_callable_points_properties" (func 0))
```
Comment (loloicci) [inline packages/vm/src/compatibility.rs:550]: This line looks not needed.

**Review by loloicci — APPROVED** [review]

**Review by loloicci — APPROVED** [review]

**Pull request merged**